source/_docs/configuration/entity-registry.markdown

```diff
@@ -55,4 +55,5 @@ value to each entry.
    
 {% linkable_title Supported Components %}
    
-At the moment, only the following components are supported: Z-Wave, Hue, Nest, LIFX, Sonos, Apple TV
+At the moment, only the following components are supported: Z-Wave, Hue, Nest, LIFX, Sonos, Apple TV, Weather Underground 
+Note that for Weather Underground, second and subsequent Personal Weather Station ID (pws_id) with have their monitored conditions suffixed with an index number. 
```

Comment:
Typo `with have` should be `will have`

Reply:
Yes, sorry.
I'm new to this.... do I have to change it myself?

Reply:
No worries - my own first few PRs were a bit of a mess :)

Yes please.